packages/instantsearch.js/src/lib/InstantSearch.ts

```diff
@@ -235,6 +252,10 @@ Use \`InstantSearch.status === "stalled"\` instead.`
       searchClient = null,
       insightsClient = null,
       onStateChange = null,
+      future = {
+        ...INSTANTSEARCH_FUTURE_DEFAULTS,
+        ...(options.future || {}),
+      },
     } = options;
 
```

Comment:
Yes we do! I'd like to do it with a link to the API reference. I don't think we solved the issue of displaying per-flavour API reference links, have we?

Reply:
I guess just the JS reference and people can switch flavours is the best we can do

Reply:
I kind of see a future where RIS/VIS would register themselves to the InstantSearch.js instance so we could generate flavoured doc URL, but it may also be overkill just for this.